src/SequenceManager.cc

```diff
@@ -327,16 +339,68 @@ void SequenceManager::rename_sequence(UniqueId unique_id, const SequenceName& ne
         throw Error(gul14::cat("Cannot rename folder ", old_path.string(),
             " to ", new_path.string(), ": ", error.message()));
     }
+
+    // commit to local repository
+    gl_.add_files({gul14::cat((const std::string&) new_path, "/*")});
+    gl_.commit(gul14::cat("Rename ", old_seq_on_disk.name.string(), " to ", new_name.string()));
```

Comment:
Here we just add files – shouldn't we also tell git to remove the old ones?

Reply:
outdated code